libs/stats/odc/stats/_cli_save_tasks.py

```diff
@@ -151,3 +162,38 @@ def gqa_predicate(ds):
     if not ok:
         # exit with error code, failure message was already printed
         sys.exit(3)
+
+
+def _parse_products(dc, products, temporal_range):
+    if len(products) == 1:
+        product = products[0]
+        dss = None
+        n_dss = None
+    elif len(products) == 2:
```

Comment:
I think this check should be in main, it kinda already there, and this should just work a sequence of products, even if that sequence has one element, it just won't be called with anything less than 2

Reply:
sorry fusing, should accept a sequence of exactly 2 elements for now at least until we generalize to more than that.

Reply:
yeah it will only accept two but its probably better that we let the fusing functions throw the errors.